src/compose/ComposeBox.js

```diff
@@ -684,8 +658,13 @@ const ComposeBox: React$AbstractComponent<Props, ImperativeHandle> = forwardRef(
           ...(!canSelectTopic && { position: 'absolute', transform: [{ scale: 0 }] }),
         },
         composeTextInput: {
+          // These border attributes override styles set in <Input />.
           borderWidth: 0,
-          borderRadius: 5,
+          borderTopLeftRadius: 5,
+          borderTopRightRadius: 5,
+          borderBottomLeftRadius: 0,
+          borderBottomRightRadius: 0,
```

Comment:
(This becomes unnecessary when there's no longer a special background for the icons row, per my previous comment.)